src/libraries/System.Threading.Overlapped/tests/ThreadPoolBoundHandle_IntegrationTests.cs

```diff
@@ -47,13 +47,14 @@ public unsafe void SingleOperationOverSingleHandle()
         Assert.Equal(DATA_SIZE, result.BytesWritten);
     }
 
-    [Fact]
+    [Theory]
+    [InlineData(nameof(MultipleOperationsOverSingleHandle))]
     [PlatformSpecific(TestPlatforms.Windows)] // ThreadPoolBoundHandle.BindHandle is not supported on Unix
-    public unsafe void MultipleOperationsOverSingleHandle()
+    public unsafe void MultipleOperationsOverSingleHandle(string tempFileNamePrefix)
```

Comment:
I can't see how this would change any behavior?

Also, if two tests in the same fixture (class) ever run concurrently with each other, I would expect it to cause all kinds of tests to break. I don't think it's a possibility.

Reply:
Oh ok, I suspected it but wasn't sure, I've undone this part of the change